os/drivers/input/touchscreen.c

```diff
@@ -88,7 +87,10 @@ static const struct file_operations g_touchdev_fileops = {
 	touch_write,	/* write */
 	NULL,			/* seek */
 	touch_ioctl,	/*ioctl */
+#if !defined(CONFIG_DISABLE_POLL)
 	touch_poll,
+#endif
+	NULL
```

Comment:
Let's leave a comment for each like other lines.

touch_poll,  /* poll */

NULL   /* unlink */